Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml

```diff
@@ -0,0 +1,879 @@
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "11"
+    scriptarguments:
+      closeNotes:
+        simple: Suspicious activity detected
+      closeReason:
+        simple: Resolved - Handled by the playbook "Suspicious LDAP search query"
+      id:
+        complex:
+          root: alert
+          accessor: id
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 530,
+          "y": 1470
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 9cd3a2cd-8887-45d7-85b9-3b1f59b4cf40
+    type: regular
+    task:
+      id: 9cd3a2cd-8887-45d7-85b9-3b1f59b4cf40
+      version: -1
+      name: Run Prevalence Check On the Actor Process
+      description: Get the prevalence of a process, identified by process_name.
+      script: '|||core-get-process-analytics-prevalence'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      process_name:
+        complex:
+          root: alert
+          accessor: osparentname
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 750,
+          "y": 260
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 81ca3384-2d21-427b-800b-cf5afc21d27f
+    type: regular
+    task:
+      id: 81ca3384-2d21-427b-800b-cf5afc21d27f
+      version: -1
+      name: Search for suspicious-related alerts by MITRE Technique
+      description: |-
+        This task searches for Cortex XSIAM suspicious alerts related to the current incident by Mitre Technique, indicating that the alert is part of an attack pattern.
+
+        Focus on identifying alerts associated with the following MITRE techniques:
+        - Any Agent Alerts within this incident.
+        - T1059 - Command and Scripting Interpreter.
+      scriptName: SearchIncidentsV2
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "15"
+    scriptarguments:
+      query:
+        complex:
+          root: alert
+          accessor: parentXDRIncident
+          transformers:
+          - operator: Cut
+            args:
+              delimiter:
+                value:
+                  simple: '-'
+              fields:
+                value:
+                  simple: "2"
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: '((mitreattcktechnique:*T1509* or (sourceBrand:TRAPS and categoryname:Malware)) and caseid:'
```

Comment:
Didn't we said yesterday we will check on 2 sub categories?

Reply:
no 
checks for agent alerts + the Command and Scripting technique only